src/content/docs/infrastructure/host-integrations/host-integrations-list/sonarqube-monitoring-integration.mdx

```diff
@@ -9,7 +9,7 @@ freshnessValidatedDate: 2023-06-28
 ---
 import SonarQubeDashboard from 'images/infrastructure_screenshot_full-sonarqube-dashboard.webp'
 
-Our SonarQube integration monitors the performance of your SonarQube application, helping you diagnose and optimize the code. Our SonarQube integration makes use of our infrastructure agent, PosgreSQL agent, NRI-Prometheus, and NRI-JMX and gives you a pre-built dashboard with your most important SonarQube metrics.
+Our SonarQube integration monitors the performance of your SonarQube application, helping you diagnose and optimize the code. Our SonarQube integration makes use of our infrastructure agent, PosgreSQL integration, NRI-Prometheus, and NRI-JMX and gives you a pre-built dashboard with your most important SonarQube metrics.
```

Comment:
Can we get a firmer verb than "makes use of". 

- Does it put data collected by these integrations in context of each other _through_ the dashboard?
- Maybe a one sentence explanation about why these other integrations are dependencies (am i using that right) for SQ? 
- Annnnd this might be out of scope but maybe identifying who finds this most useful can also be helpful. Mostly b/c asking customers to install 4 different technologies is a huge ask, so helping them decide if it's useful for their use case/persona can make the value prop more evident upfront